clang/lib/Basic/Targets/RISCV.cpp

```diff
@@ -257,7 +257,7 @@ bool RISCVTargetInfo::initFeatureMap(
 
   // If a target attribute specified a full arch string, override all the ISA
   // extension target features.
-  const auto I = llvm::find(FeaturesVec, "__RISCV_TargetAttrNeedOverride");
+  const auto I = llvm::find(FeaturesVec, "+__RISCV_TargetAttrNeedOverride");
```

Comment:
Where does this thing come from in the first place? AFAICT there is no equivalent for other targets, and having some random placeholder feature seems rather ugly, so why is RISC-V special?

Reply:
This feature was used to implement the target attribute override mechanism.

I think it could be removed after replacing it with several negative target features. Here is the pull request for this change: https://github.com/llvm/llvm-project/pull/106680